assets/blocks/course-outline/course-block/block.json

```diff
@@ -12,6 +12,42 @@
     "animationsEnabled": {
       "type": "boolean",
       "default": true
+    },
+    "borderColor": {
+      "type": "string"
+    },
+    "customBorderColor": {
+      "type": "string"
+    }
+  },
+  "styles": [
+    {
+      "name": "borderless",
+      "label": "Borderless"
+    },
+    {
+      "name": "default",
+      "label": "Bordered",
+      "isDefault": true
     }
+  ],
+  "example": {
```

Comment:
A block with these example attributes are also created for preview on the Block Style settings. (On addition to showing the example when inserting from the block library)

Reply:
Hum! Very interesting! TIL!